clang/test/dpct/asm/selp.cu

```diff
@@ -0,0 +1,55 @@
+// UNSUPPORTED: cuda-8.0, cuda-9.0, cuda-9.1, cuda-9.2, cuda-10.0, cuda-10.1, cuda-10.2
+// UNSUPPORTED: v8.0, v9.0, v9.1, v9.2, v10.0, v10.1, v10.2
+// RUN: dpct --format-range=none -out-root %T/selp %s --cuda-include-path="%cuda-path/include" -- -std=c++14 -x cuda --cuda-host-only
+// RUN: FileCheck %s --match-full-lines --input-file %T/selp/selp.dp.cpp
+// RUN: %if build_lit %{icpx -c -fsycl %T/selp/selp.dp.cpp -o %T/selp/selp.dp.o %}
+
+// clang-format off
+#include <cuda_runtime.h>
+#include <cstdint>
+
+__global__ void selp() {
+  int16_t i16;
+  uint16_t u16;
+  int32_t i32;
+  uint32_t u32;
+  int64_t i64;
+  uint64_t u64;
+  float f32;
+  double f64;
+
+  // CHECK: i16 = i16 == 1 ? i16 : i16;
+  asm("selp.s16 %0, %1, %2, %3;" : "=h"(i16) : "h"(i16), "h"(i16), "h"(i16));
+  
+  // CHECK: u16 = u16 == 1 ? u16 : u16;
+  asm("selp.u16 %0, %1, %2, %3;" : "=h"(u16) : "h"(u16), "h"(u16), "h"(u16));
```

Comment:
better to update the arg names to make sure the arg list in migrated code is correct.

Reply:
done